scripts/sell_unused_slots.py

```diff
@@ -0,0 +1,125 @@
+from datetime import (
+    datetime,
+    timedelta,
+)
+import sys
+from typing import (
+    Dict,
+    Iterable,
+    List,
+)
+
+from azul import (
+    cached_property,
+    logging,
+)
+from azul.bigquery_reservation import (
+    SlotManager,
+)
+from azul.deployment import (
+    aws,
+)
+from azul.logging import (
+    configure_script_logging,
+)
+from azul.modules import (
+    load_app_module,
+)
+
+log = logging.getLogger(__name__)
+
+
+class ReindexDetector:
+    # Minutes
+    interval = 5
+
+    # Minimum number of contribution Lambda invocations per interval for a
+    # reindexing to be considered active
+    threshold = 0
+
+    @cached_property
+    def _cloudwatch(self):
+        return aws.client('cloudwatch')
+
+    @cached_property
+    def _lambda(self):
+        return aws.client('lambda')
+
+    @cached_property
+    def contribution_lambda_suffix(self) -> str:
+        sep = '-'
```

Comment:
Do not like hard-coding the separator here. There is functionality for qualifying resource names in `azul.Config`.